public/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "public",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "scss": "node-sass --watch assets/sass/main.scss -o assets/css"
```

Comment:
This is a development script. We need a script the compiles the scss files without watch.